cypress/e2e/circulation-log/verify-user-barcode-shown-in-request-actions.cy.js

```diff
@@ -0,0 +1,112 @@
+import NewRequest from '../../support/fragments/requests/newRequest';
+import OtherSettings from '../../support/fragments/settings/circulation/otherSettings';
+import CheckOutActions from '../../support/fragments/check-out-actions/check-out-actions';
+import RequestsSearchResultsPane from '../../support/fragments/requests/requestsSearchResultsPane';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+describe('Circulation log', () => {
+  let userData = {};
+  let requestId;
+  let itemBarcode;
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances({ count: 1 }),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+  };
+
+  before('Preconditions:', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    OtherSettings.setOtherSettingsViaApi({ titleLevelRequestsFeatureEnabled: true });
+    cy.createTempUser([
+      Permissions.uiRequestsCreate.gui,
+      Permissions.uiRequestsView.gui,
+      Permissions.uiRequestsEdit.gui,
+      Permissions.requestsAll.gui,
+      Permissions.checkoutAll.gui,
+      Permissions.circulationLogAll.gui,
+    ]).then((userProperties) => {
+      userData = userProperties;
+      UserEdit.addServicePointsViaApi(
+        [testData.servicePoint.id],
+        userData.userId,
+        testData.servicePoint.id,
+      );
+      itemBarcode = testData.folioInstances[0].barcodes[0];
+    });
+  });
+
+  after('Deleting created entities', () => {
+    CheckInActions.checkinItemViaApi({
+      itemBarcode,
+      servicePointId: testData.servicePoint.id,
+      checkInDate: new Date().toISOString(),
+    });
+    Requests.deleteRequestViaApi(requestId);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [testData.servicePoint.id]);
+    ServicePoints.deleteViaApi(testData.servicePoint.id);
+    Users.deleteViaApi(userData.userId);
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(
+      testData.folioInstances[0].barcodes[0],
+    );
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      testData.defaultLocation.institutionId,
+      testData.defaultLocation.campusId,
+      testData.defaultLocation.libraryId,
+      testData.defaultLocation.id,
+    );
+  });
+
+  it(
+    'C360553 Verify that user barcodes shown in request actions (Volaris) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.volaris] },
+    () => {
+      cy.login(userData.username, userData.password, {
```

Comment:
Please move 'cy.login' to 'cy.createTempUser' in preconditions

Reply:
done